source/common/config/grpc_mux_impl.cc

```diff
@@ -16,6 +16,8 @@ GrpcMuxImpl::GrpcMuxImpl(const envoy::api::v2::core::Node& node, Grpc::AsyncClie
     : node_(node), async_client_(std::move(async_client)), service_method_(service_method),
       time_source_(time_source) {
   retry_timer_ = dispatcher.createTimer([this]() -> void { establishNewStream(); });
+  backoff_strategy_ptr_ = std::make_unique<ExponentialBackOffStrategy>(
```

Comment:
Does this affect any existing tests? If not, how come?

Reply:
It did not actually.